corehq/apps/app_manager/templates/app_manager/partials/case_config_advanced.html

```diff
@@ -79,23 +87,38 @@
 
             <!-- ko if:  actionType === 'open' -->
             <div class="control-group" data-bind="visible: allow_subcase, css: {warning: validate_subcase()}">
-                <label class="checkbox">
-                    <input type="checkbox" data-bind="checked: subcase"/>
-                    {% trans "Make this case a subcase" %}
-                    <span data-bind="if: subcase">
-                        {% trans " of " %}
-                    </span>
-                </label>
-                <span data-bind="if: subcase">
-                    <select data-bind="optstr: $parent.getCaseTags('subcase', $data),
-                        value: parent_tag">
+
+                <span style="font-weight: bold;">Parent Indices</span>
+                <ul data-bind="foreach: case_indices">
+                    <li>
+                        Parent Tag: <span data-bind="text: tag"></span><br />
+                        Reference ID: <span data-bind="text: reference_id"></span><br />
+                        Relationship: <span data-bind="text: relationship"></span><br />
+                        <i class="icon-trash"></i>
+                        <a href="#" data-bind="click: $parent.removeCaseIndex">Remove index</a>
+                    </li>
+                </ul>
+
+                {% trans "Make this case a subcase of " %}
+                <span data-bind="with: newCaseIndex">
+                    <select data-bind="optstr: $parents[1].getCaseTags('subcase', $parent),
+                        value: tag">
                     </select>
+                    <br />
+
                     {% if show_custom_ref %}
-                    <div>
-                        <label>{% trans "Parent reference id: " %}</label>
-                        <input type="text" data-bind="value: parent_reference_id"/>
-                    </div>
+                    <label>{% trans "Parent reference ID:" %}</label>
+                    <input type="text" data-bind="value: reference_id"/>
+                    <br />
                     {% endif %}
+
+                    <label>{% trans "Relationship:" %}</label>
+                    <select data-bind="options: $parent.relationshipTypes,
+                                       optionsCaption: 'Child case or extension case?',
+                                       value: relationship"></select>
+                    <br />
+
+                    <button data-bind="click: $parent.addCaseIndex">Add</button>
```

Comment:
Thanks. I'll check that out. I couldn't push it and make a new instance because I bind newCaseIndex (https://github.com/dimagi/commcare-hq/blob/extension_cases/corehq/apps/app_manager/templates/app_manager/partials/case_config_advanced.html#L103-103). If I don't bind it, it might work.